### ISSUE TITLE
list.tsx throws error 

### ISSUE THREAD

**Issue by thomashoddinott** [bmvantunes/youtube-2020-march-nextjs-part3#3]
https://github.com/bmvantunes/youtube-2020-march-nextjs-part3/blob/ce3ed5c0d24985f19ae45fe406f9bd3f71cf0cf1/src/pages/List.tsx#L15-L20

Because `<MyComp>` is between the `<Link>` tags it will throw this error: 

```
React.Children.only expected to receive a single React element child.
```

It looks like `MyComp` shouldn't be here though? We just print 'Hello' after each link. If we require it we can move it outside the `Link` tags

```
          <Link as={`/${e.vehicle}/${e.ownerName}`} href="/[vehicle]/[person]">
            <a>
              Navigate to {e.ownerName}'s {e.vehicle}
            </a>
          </Link>
        <MyComp/>
```

 and `http://127.0.0.1:3000/list` will generate:

```
Navigate to Bruno Antunes's Car
Hello
Navigate to Bruno Antunes's Bike
Hello
Navigate to John Doe's Bike
Hello
Navigate to Bill Gates's Airplane
Hello
Navigate to Elon Musk's SpaceX
Hello
```

**Comment by bmvantunes** [comment]
Thank you very much for the report =D

Yes until recently, I was recording and after recording, I was typing the code again and committing. I'm sure I was playing with something and forgot that MyComp there eheh

I already removed it from there and committed =)

**Comment by thomashoddinott** [comment]
No worries! I am completely new to web dev so https://github.com/bmvantunes/youtube-2020-march-nextjs-part3/commit/53655406a35bcf63a986adf725d830d5a76d9043 might just clear up any confusion for others like me who come from the YT video to see the code here.

Cheers

**Comment by bmvantunes** [comment]
I really appreciate if you find something that differs from the videos or doesn't work at all. It's highly appreciated =)

Only recently I started to do breaks on the video to add commits. Before that, I was "repeating" the process off-camera and committing - which can lead to differences.

If you see part 8 I started to stop the video specifically to add commits - that way what's in the repo is 100% what's on video, step by step =)
